doc/03-omnifest/01-directive.md

```diff
@@ -200,7 +200,7 @@ otk.meta.kiwi:
 
 External directives. Directives starting with `otk.external` are redirected
 to `/usr/libexec/otk/`-binaries. For example the directive
-`otk.external.osbuild.depsolve_dnf4` will execute `otk-osbuild depsolve_dnf4`
+`otk.external.osbuild_depsolve_dnf4` will execute `otk_external_osbuild_depsolve_dnf4`
```

Comment:
Slightly silly question - there is nothing here that forces us to use names like `osbuild_` we could keep the previous `osbuild.` as long as we name the executable files accordingly?

Reply:
We could.